libyul/AsmPrinter.cpp

```diff
@@ -258,6 +258,12 @@ string AsmPrinter::operator()(Block const& _block) const
 	}
 }
 
+std::string AsmPrinter::operator()(Comment const& _comment) const
+{
+	// Leading newline for better readability of comments.
+	return "\n// " + _comment.text;
```

Comment:
Please check that it does not contain a newline.

Reply:
Actually, I did that at first, but found out that the text (in my tests) didn't even contain trailing newline, that's why I dropped my extra 3 lines for L/R trimming. I can add them again. Thanks :-)

Reply:
I mean `_comment.text` cannot contain a newline, otherwise the comment would end at a place where it should not.

Reply:
me too :)